CONTRIBUTING.md

```diff
@@ -54,16 +54,18 @@ contributions via GitHub pull requests. When contributing code please do the
 following:
 
 * Fork the repository and make your changes in a feature branch.
+* Please include new or update existing PlantUML sequence/architecture diagrams
+  that relate to your PR. ([/docs/diagrams/src/readme.md](details) on how we
```

Comment:
When I click on the link here it leads me to a 404 page not found.

Reply:
oh, hmm. I guess we can't have links to other pages in the repo. I'll just leave it as a filepath then